WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/LoginJetpackSetupViewModel.swift

```diff
@@ -167,9 +175,10 @@ private extension LoginJetpackSetupViewModel {
             guard let self else { return }
             switch result {
             case .success:
+                self.analytics.track(.loginJetpackSetupScreenInstallSuccessful)
                 self.activateJetpack()
             case .failure(let error):
-                // TODO: add tracks
+                self.analytics.track(.loginJetpackSetupScreenInstallFailed)
```

Comment:
I missed this - we should probably track the error as well:
```suggestion
                self.analytics.track(.loginJetpackSetupScreenInstallFailed, withError: error)
```

Reply:
🤦 

Done in https://github.com/woocommerce/woocommerce-ios/pull/8187/commits/715348acb5a2fcc4dadeff794b5663fdf39dd57b